src/main/java/seedu/address/ui/CountryNoteListPanel.java

```diff
@@ -32,6 +33,19 @@ public CountryNoteListPanel(ObservableList<CountryNote> countryNoteObservableLis
         countryNoteListView.setCellFactory(listView -> new CountryListViewCell());
     }
 
+    /**
+     * Sets the header of the CountryNoteListPanel.
+     *
+     * @param country The country to be displayed.
+     */
+    public void setHeader(Country country) {
+        if (country == null) {
```

Comment:
Should this be country.equals(Country.getNullCountry())?

Reply:
Yep